src/dev/i18n_tools/bin/run_i18n_check.ts

```diff
@@ -131,13 +128,15 @@ run(
       {
         concurrent: false,
         exitOnError: true,
-        renderer: process.env.CI ? 'verbose' : ('default' as any),
+        forceTTY: false,
+        renderer:
```

Comment:
Was the silent flag still needed if we just removed verbose logging?

Reply:
There's a similar discussion where I've lifted this change from: https://github.com/elastic/kibana/pull/194379#discussion_r1816697028

My current interpretation is this:
 - If we only set the `renderer` to `undefined` or `default` it will still output a huge amount of output (like how we currently have it on CI)
 - We can channel in the `--quiet`/`--silent` flags, and set the `renderer='silent'` to quiet all task-related logs, and leave the ToolingLog to decide on logging whatever is needed.